miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridgeDispatcher.kt

```diff
@@ -18,7 +18,7 @@ interface UserInfoBridgeDispatcher {
         onSuccess: (userName: String) -> Unit,
         onError: (message: String) -> Unit
     ) {
-        throw MiniAppSdkException("The `UserInfoBridgeDispatcher.getUserName` $NO_IMPL")
+        throw MiniAppSdkException(NO_IMPL)
```

Comment:
I think, the reason of previous message is to tell the HostApp which method should be implemented. Now, it's difficult to understand which one should be implemented using this line.

Reply:
The hostapp never catch this error message because we catch it and send the error to miniapp via bridge js.